os/arch/arm/src/bcm4390x/Kconfig

```diff
@@ -0,0 +1,161 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the TinyAra tools repository.
```

Comment:
We don't have tools repository which is in Nuttx. Please use below:
```
#
# For a description of the syntax of this configuration file,
# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
#
```

Reply:
it will be changed as your guidance.

BR, HJ Lee.